src/test/features/signUp.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Sign up
+
+        Background:
+            Given  User passes the authorization on "Auth Page"
+            
+        Scenario: Create a new user
+             Then User clicks "Get Started Button" on "Landing Page"
+              And User clicks "Register with Email Button" on "Landing Page"
+              And User fills "Company Name Input" as "Test Company" on "Sign Up Page"
+              And User fills "Phone Number Input" as "+12345678" on "Sign Up Page"
+              And User fills "Company Details Input" as "Test Company" on "Sign Up Page"
+              And User fills "Account Details Input" as "Test Account" on "Sign Up Page"
+              And User fills "Full Name Input" as "Test Name" on "Sign Up Page"
+              And User fills "Company Email Input" as "ilhomislamov38" on "Sign Up Page"
+              And User fills "Confirm Email Input" as "ilhomislamov38" on "Sign Up Page"
+              And User chooses "Country" as "India" from "Country List" on "Sign Up Page"
```

Comment:
How about: `User chooses "India" from "Country list dropdown" on "Sign up page"` ? Does it look simpler?